doc/mgmt/SONiC_Management_Framework_sFlow.md

```diff
@@ -133,9 +140,38 @@ POST - Change existing sFlow configuration in CONFIG DB
 # 9 Unit Test
 The unit-test for this feature will include:
 1. configuration via CLI
+
+| Test Name | Test Description |
+| :------ | :----- |
+| Enable sflow | Verfiy sFlow is enabled in configDB |
+| Disable sflow | Verify sFlow is disabled in configDB |
+| Configure polling-interval | Verify sFlow polling-interval is set in configDB |
+| Disable polling-interval | Verify sflow polling-interval is removed from configDB (back to default)
+| Add a collector | Verify a collector has been added into configDB |
+| Add a collector with port # | Verify a collector has been added into configDB with user supplied port #|
+| Delete a collector | Verify a collector has been deleted from configDB
+| Enable sflow on interface | Verfiy sFlow for an interface is enabled in configDB |
+| Disable sflow on interface | Verfiy sFlow for an interface is disabled in configDB |
+| Configure sampling-rate on interface | Verify sampling-rate for an interface is set in configDB|
+| Disable sampling-rate on interface | Verify sampling-rate has returned to default
+
 2. show sflow configuration via CLI
+
 3. configuration (POST) via GNMI
```

Comment:
Please use gNMI instead of GNMI throughout the doc

Reply:
+1

Reply:
The version you're reading is OUTDATED.